week2/homework/exercises/1-blog-API/server.js

```diff
@@ -1,10 +1,53 @@
 const express = require('express')
 const app = express();
- 
+const fs = require("fs");
 
-// YOUR CODE GOES IN HERE
-app.get('/', function (req, res) {
-  res.send('Hello World')
+app.use(express.json())
+
+
+app.post('/blogs', (req, res) => {
+  let title = req.body.title
+  let content = req.body.content
+  fs.writeFileSync(title, content)
+  res.end('ok')
+})
+
+app.put('/posts/:title', (req, res) => {
+  let title = req.body.title
+  console.log(title)
+  let content = req.body.content
+  if (fs.existsSync(title) || fs.existsSync(content)) {
```

Comment:
```suggestion
  if (fs.existsSync(title)) {
```
no need to check the content, that will not cause any problems. We check the title because the operating system does not allow two files with the same name  to exist in the same folder.